src/pages/EmployeeProfile/data/mockData.js

```diff
@@ -0,0 +1,34 @@
+const person1 = {
```

Comment:
If I just pass const person1 = {}; it results in this:
<img width="307" alt="Screenshot 2024-01-17 at 16 53 43" src="https://github.com/SE2-WS23/Skillverse-Web/assets/136974808/5076e57a-ccb7-4468-86b9-cd9a20ac58ef">